docs/docs/sourcing-from-drupal.md

```diff
@@ -13,45 +13,46 @@ It only takes a few steps to use Gatsby with Drupal as a headless CMS (also know
 
 ### How to implement Drupal + Gatsby
 
-Hooking up Gatsby to a new or existing Drupal site takes just a few steps:
+Hooking up Gatsby to a new or existing Drupal site is simple.
 
```

Comment:
@rahulkumaran one more idea! Would you add a section here titled ## Quick Start that includes this text?
Will you retitle this section "Quick start" and instead of giving instructions, say something like:
"This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [Quick Start guide](/docs/), then come back."

I'm realizing this will save space and make sure we can maintain the Quick Start guide in just one place. Thanks for your patience! I am thinking of new ideas as more contributors start adding to the docs! :)

Reply:
@shannonbux I've left the heading as it is, but I've added a sub heading `Quick Start` with the info you mentioned above. Hope it's good! :smile: